package-lock.json

```diff
@@ -14,34 +14,29 @@
       }
     },
     "@babel/core": {
-      "version": "7.8.7",
```

Comment:
It looks like you accidentally updated package-lock not sure how though. There are no changes in `package.json`. I would recommend restoring this to the current `package-lock.json` from master by running `git fetch origin && git checkout origin/master -- package-lock.json`.